test/built-ins/Array/from/mapfn-attached-typedarray.js

```diff
@@ -0,0 +1,25 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: mapFn is attached to the TypedArray itself

```

Comment:
I don't understand this test. The file name and description reference TypedArray, but that intrinsic is not used in the test body. Also, the modification of the ArrayPrototype built-in doesn't seem necessary.